elasticsearch/elasticsearch.d.ts

```diff
@@ -125,6 +129,17 @@ declare module Elasticsearch {
         body: string | any;
     }
 
+    export interface IndicesPutMappingParams extends GenericParams {
+        timeout?: Date | number;
+        masterTimeout?: Date | number;
+        ignoreUnavailable?: boolean;
+        allowNoIndices?: boolean;
+        expandWildcards?: string;
```

Comment:
Change it so it reflects the only options that exists.. 
expandWildcards?: "open" | "closed" | "none" | "all";